platform/ext/target/nordic_nrf/nrf5340/gcc/startup_nrf5340_s.S

```diff
@@ -244,6 +244,12 @@ Reset_Handler:
 #endif /* __STARTUP_CLEAR_BSS_MULTIPLE || __STARTUP_CLEAR_BSS */
 
     cpsid   i              /* Disable IRQs */
+
+/* Setup Vector Table Offset Register. */
+    ldr     r0, =__Vectors
+    ldr     r1, =#0xE000ED08 /* SCB->VTOR */
```

Comment:
I think the "#" needs to be omitted (pseudo-ldr does not need it,  i think). 
MOV instructions need #.

Reply:
You can define the VTOR as a constant. See an example from Zehyr:
```

    ldr v4, =_SCS_ICSR
    ldr v3, =_SCS_ICSR_UNPENDSV

```

...where the macros are defined (for ASM)

Reply:
I know. But I didn't want to introduce new definitions just for this one line.
I just used `SCB->VTOR` as the comment to make it clear what is happening here and to make it possible to easily locate the place where VTOR is initialized for nRF targets (just by grepping for `SCB->VTOR`).